src/convertToObject.js

```diff
@@ -6,7 +6,14 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .filter((style) => style.trim() !== '')
+    .map((style) => style.split(':'));
+
+  return styles.reduce((prevStyles, [property, value]) => {
+    return { ...prevStyles, [property.trim()]: value.trim() };
+  }, {});
```

Comment:
1) This is not previous value. This is `accumulator`
2) use destructuring separately for better readability
3) u don't need create object on each iteration.
at each iteration you can add keys to the already created object (accumulator (second parameter)).

```suggestion
  return styles.reduce((acc, style) => { //1
    const [property, value] = style; // 2
    
    acc[property.trim()] = value.trim(); //3
  
    return acc;
  }, {});
```